GPU monitoring script for Windows version (#1)
GoogleCloudPlatform/compute-gpu-monitoring: windows/README.md

# Google Compute Engine GPU Monitoring (Windows)

This repository hosts an example script that
can be used to monitor GPU utilization parameters
on GCE instances. 

## Requirements

To run this script you need to meet the following criteria:

* It can be run only on Google Compute Engine machines.
* It requires the `nvidia-smi` tool to be properly installed in the default folder of C:\Program Files\NVIDIA Corporation\NVSMI
* You need to have Cloud Monitoring dashboard created. This is automatically done on your first visit on the [Cloud Monitoring page](https://console.cloud.google.com/monitoring) in the Cloud Console.

The `nvidia-smi` tool is installed by default if you follow the
driver installation instructions in our 
[public documentation](https://cloud.google.com/compute/docs/gpus/install-drivers-gpu).

## Installation

This instruction assumes installation in `C:\Program Files\NVIDIA Corporation` directory.

> If you change the directory the `nvidia-smi` tool is installed, you have to update your script to be downloaded.

Reviewer: ```suggestion
If you change the directory the `nvidia-smi` tool is in installed, you have to update your script to use this new directory.
```
